packages/notification-center/src/components/notification-center-component.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { FunctionComponent } from 'react';
+import { IMessage, MessageActionStatusEnum, ButtonTypeEnum } from '@novu/shared';
+
+import { INovuProviderProps, NovuProvider } from './novu-provider';
+import { IPopoverNotificationCenterProps, PopoverNotificationCenter } from './popover-notification-center';
+import { NotificationBell } from './notification-bell';
+import { useNotifications } from '../hooks';
+import { reactToWebComponent } from '../utils';
+import { ColorScheme } from '../shared/config/colors';
+
+type NovuProviderProps = Pick<
+  INovuProviderProps,
+  | 'onLoad'
+  | 'stores'
+  | 'backendUrl'
+  | 'socketUrl'
+  | 'subscriberId'
+  | 'applicationIdentifier'
+  | 'subscriberHash'
+  | 'i18n'
```

Comment:
I would create a union with the `SelectedProviderProps`.

Reply:
not sure if I get your idea :)

Reply:
Sorry, I would extract that union into a union type. That was a name suggestion.

Reply:
ohh got it, thanks will do it!